packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -77,42 +77,24 @@ export class W3mAccountDefaultWidget extends LitElement {
     }
 
     const networkImage = AssetUtil.getNetworkImage(this.network)
+    const account = AccountController.state.allAccounts?.find(acc => acc.address === this.address)
+    const label = AccountController.state.addressLabels.get(this.address)
 
     return html`<wui-flex
         flexDirection="column"
         .padding=${['0', 'xl', 'm', 'xl'] as const}
         alignItems="center"
         gap="l"
       >
-        <wui-avatar
-          alt=${ifDefined(this.address)}
+        <wui-profile-button-v2
```

Comment:
Why don't we refactor the current `wui-profile-button` since it's only used here? We'll be having two different versions unnecessarily. Bc also the `*-v2` naming not quite good/descriptive.

Reply:
I would leave this to be addressed at a later point such as tech debt week